### PR TITLE
Fixes Ripley Drills

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
rscadd: Ripley drills work again. Go tear up lavaland.
/:cl:

### CODE REVIEW

File: code/game/mecha/equipment/tools/mining_tools.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 /obj/item/mecha_parts/mecha_equipment/drill/action(atom/target)
 	if(!action_checks(target))
 		return
-	if(istype(target, /turf) && !istype(target, /turf))
```
Comment (xxalpha) [inline code/game/mecha/equipment/tools/mining_tools.dm]: what the fuck is this

Comment (Shadowlight213) [inline code/game/mecha/equipment/tools/mining_tools.dm]: @duncathan 10/10 coding

Comment (Supermichael777) [inline code/game/mecha/equipment/tools/mining_tools.dm]: this is the magic of find replace spam

Comment (duncathan) [inline code/game/mecha/equipment/tools/mining_tools.dm]: blame whoever wrote `if(istype(target, /turf) && !istype(target, /turf/simulated)) ` in the first place

**Pull request merged**